packages/gasket-plugin-elastic-apm/README.md

```diff
@@ -99,6 +99,41 @@ According to the [Elastic APM docs], the _Elastic APM agent for Node.js is a
 singleton_. This means that you can require and configure singleton in various
 hooks of your Gasket app, such as with the [init] or [middleware] lifecycles.
 
+## Lifecycles
+
+### transactionName
```

Comment:
Are `transactionName` and `transactionLabel` a little too generic for a lifecycle names? What about prefixing with _apm_ or perhaps _elastic_, i.e. `apmTransactionName`, or `elasticTransactionName`?

Reply:
I was wondering if we wanted to keep the lifecycle names more generic in case we have other APM providers we want to support later.

Reply:
Merged together into an `apmTransaction` lifecycle.